resources/vvv-custom.yml

```diff
@@ -25,8 +25,18 @@ sites:
   # The following commented out site configuration will create a environment useful
   # for contributions to the WordPress meta team:
   
-  wordpress-meta-environment:
-    repo: https://github.com/WordPress/meta-environment.git
+ wordpress-meta-environment:
```

Comment:
did you keep the space counts? alignment WILL be a problem here

Reply:
oops, lemme redo that, copy pasta error from the readme